powerauth-java-http/src/main/java/io/getlime/security/powerauth/http/validator/ValueTypeValidator.java

```diff
@@ -52,7 +52,7 @@ public class ValueTypeValidator {
      * Admissible protocol versions in the header.
      */
     private static final Set<String> PROTOCOL_VERSIONS = new HashSet<>(Arrays.asList(
-            "3.1", "3.0", "2.1", "2.0"
+            "3.1", "3.0"
```

Comment:
Nitpick: If we touch this line, we may use JDK 17+, making it also immutable.
```suggestion
private static final Set<String> PROTOCOL_VERSIONS = Set.of("3.1", "3.0");
```